ajax/libs/stupidtable/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "stupidtable",
   "filename": "stupidtable.min.js",
-  "version": "0.0.1",
+  "version": "1.0.2",
   "description": "A stupidly small and simple jQuery table sorter plugin",
-  "homepage": "http://joequery.github.io/Stupid-Table-Plugin/",
+  "homepage": "https://joequery.github.io/Stupid-Table-Plugin/",
```

Comment:
@sufuf3 It seemed you combine package.json enrichment with adding auto-update config. I think you should notify package.json enrichment in commit message, or separate it into 2 commits [ref](https://github.com/cdnjs/cdnjs/pull/10014/commits). What do you think?